### ISSUE TITLE
-[NSNull length]: unrecognized selector sent to instance

### ISSUE THREAD

**Issue by albeebe** [Instabug/Instabug-iOS#52]
After i submit feedback for the **very first time**, the app crashes with

```*** Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '-[NSNull length]: unrecognized selector sent to instance 0x1a0c6cea8'```

If i restart the app, it instantly crashes.
If i comment out the Instabug startWithToken and restart the app everything is fine.

Heres my initializer at the very bottom of `didFinishLaunchingWithOptions`

```Instabug.startWithToken("TOKEN_GOES_HERE", invocationEvent: IBGInvocationEventFloatingButton)
Instabug.setFloatingButtonEdge(CGRectEdge.MinXEdge, withTopOffset: 100)```

The feedback i report does show up in the dashboard at instabug.com

XCode 7.2

**Comment by albeebe** [comment]
Symbolicated crash log http://pastebin.com/CZv907J4

**Comment by m619** [comment]
Hi @albeebe! Thanks for the symobolicated crash log, made our lives easier. :D
Would you please try the attached build and see if it fixes your issue?
[InstabugParsingCrashFix.zip](https://github.com/Instabug/Instabug-iOS/files/116471/InstabugParsingCrashFix.zip)

**Comment by albeebe** [comment]
@m619 Excellent, it appears to be fixed!

I got the app into the state where it would instantly crash every time the app was launched. I replaced Instabug with the build you posted. Recompiled, reopened, no crash! Tried a bunch of things to get it to crash again but so far its solid.

Very much appreciate your quickness with this bug. Your product is exactly what we needed and we look forward to doing business with you!
